inspec/a2-iam-v2p1-only-integration/controls/get_authorized_projects.rb

```diff
@@ -0,0 +1,120 @@
+# encoding: utf-8
+# copyright: 2019, Chef Software, Inc.
+# license: All rights reserved
+
+title 'IAM v2.1 get all allowed projects for global projects filter'
+
+control 'iam-v2-global-projects-filter-1' do
+  title 'getting all allowed projects for a user on v2.1'
+
+  PROJECT_ID_1 = "inspec-custom-project-1-#{Time.now.utc.to_i}"
+  PROJECT_ID_2 = "inspec-custom-project-2-#{Time.now.utc.to_i}"
+  PROJECT_ID_3 = "inspec-custom-project-3-#{Time.now.utc.to_i}"
+
+  PROJECT_1 = {
+          id: PROJECT_ID_1,
+          name: "Test Project 1"
+  }
+  PROJECT_2 = {
+          id: PROJECT_ID_2,
+          name: "Test Project 2"
+  }
+  PROJECT_3 = {
+          id: PROJECT_ID_3,
+          name: "Test Project 3"
+  }
+
+  PROJECTS = [ PROJECT_1, PROJECT_2, PROJECT_3 ]
+  
+  describe 'allowed projects' do
+    before(:all) do
+      PROJECTS.each do|project|
+        resp = automate_api_request("/apis/iam/v2beta/projects",
+          http_method: 'POST',
+          request_body: project.to_json
+        )
+        expect(resp.http_status).to eq 200
+      end
+    end
+    
+    after(:all) do
+      PROJECTS.each do|project|
+        resp = automate_api_request("/apis/iam/v2beta/projects/#{project[:id]}", http_method: 'delete')
```

Comment:
Suggest capitalizing http methods 'delete' here and 'get' below per usual convention.
You do capitalize 'POST' in this file, and another instance of 'DELETE'.